app/code/Transoft/Blog/Api/BlogRepositoryInterface.php

```diff
@@ -0,0 +1,63 @@
+<?php
+declare(strict_types=1);
+
+namespace Transoft\Blog\Api;
+
+use Magento\Framework\Api\SearchCriteriaInterface;
+use Magento\Framework\Api\SearchResultsInterface;
+use Magento\Framework\Exception\CouldNotDeleteException;
+use Magento\Framework\Exception\InputException;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\Framework\Exception\TemporaryState\CouldNotSaveException;
+use Transoft\Blog\Api\Data\BlogInterface;
+
+interface BlogRepositoryInterface
```

Comment:
Please provide DockBlock. Refer to https://devdocs.magento.com/guides/v2.3/coding-standards/docblock-standard-general.html#classes-interfaces